src/engine/test/source/base/utils/wazuhProtocol/wazuhRequest_test.cpp

```diff
@@ -4,6 +4,55 @@
 #include <json/json.hpp>
 #include <logging/logging.hpp>
 
+void initLogging(void)
+{
+    // Logging setup
+    logging::LoggingConfig logConfig;
+    logConfig.logLevel = "off";
+    logConfig.filePath = logging::DEFAULT_TESTS_LOG_PATH;
+    logging::loggingInit(logConfig);
+}
```

Comment:
Can this function be unified for all tests in a single file so as not to duplicate code, something like a base module for the tests?

Reply:
Done